### ISSUE TITLE
npm package run - Error: Cannot find module 'object-assign'

### ISSUE THREAD

**Issue by killfish** [chentsulin/electron-react-boilerplate#75]
I'm getting the following error when trying to package the sample application.

I'm running node 4.2.1 and npm 2.14.7.

```
electron-react-boilerplate|master⚡ ⇒ npm run package              

> electron-react-boilerplate@0.7.0-beta package /Users/killfish/WebstormProjects/electron-react-boilerplate
> NODE_ENV=production node package.js

module.js:339
    throw err;
    ^

Error: Cannot find module 'object-assign'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/killfish/WebstormProjects/electron-react-boilerplate/package.js:6:14)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:467:10)

npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "package"
npm ERR! node v4.2.1
npm ERR! npm  v2.14.7
npm ERR! code ELIFECYCLE
npm ERR! electron-react-boilerplate@0.7.0-beta package: `NODE_ENV=production node package.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the electron-react-boilerplate@0.7.0-beta package script 'NODE_ENV=production node package.js'.
npm ERR! This is most likely a problem with the electron-react-boilerplate package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     NODE_ENV=production node package.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls electron-react-boilerplate
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/killfish/WebstormProjects/electron-react-boilerplate/npm-debug.log
```

**Comment by killfish** [comment]
I was able to fix the issue by installing the package myself with ```npm i object-assign --save-dev```. There is either an issue with your package.json or with npm not installing all of module dependencies.

**Comment by chentsulin** [comment]
Fixed via #76.
